src/controllers/playback/video/index.js

```diff
@@ -1340,6 +1340,12 @@ export default function (view) {
             case 'PageDown':
                 playbackManager.previousChapter(currentPlayer);
                 break;
+            case 'g':
```

Comment:
I think it make sense that CapsLock prevent the actions, using the capitals would also capture shift+g shift+h that could bind to different actions, e.g. in this case 1sec increase instead of 0.1 . Not sure what are the convention for keybinding and CapLocks (i.e. CapsLock ON , shift+f is f or F)

Reply:
> I think it make sense that CapsLock prevent the actions, using the capitals would also capture shift+g shift+h that could bind to different actions, e.g. in this case 1sec increase instead of 0.1 .

Yes, `G` will capture `Shift+G`. We can ignore this fact for now or add `if (!e.shiftKey) ...` _(Actually, we need normal binding manager that will allow us to customize bindings)_.
But we should not rely on upper/lower case. Technically, we need keyboard button code instead of character/symbol.


If we want to alter the action, we should do it as
https://github.com/jellyfin/jellyfin-web/blob/0e3f6fa77b8872943ef495e9d488a3e04750397e/src/controllers/playback/video/index.js#L1279-L1284

> Not sure what are the convention for keybinding and CapLocks (i.e. CapsLock ON , shift+f is f or F)

It has no effect in VSCode, YouTube, Firefox: for example, `Ctrl+O` w/wo CapsLock.